src/lib.rs

```diff
@@ -250,7 +255,6 @@ pub struct Receiver<T> {
 
 unsafe impl<T: Send> Send for Sender<T> {}
 unsafe impl<T: Send> Send for Receiver<T> {}
-impl<T> Unpin for Receiver<T> {}
```

Comment:
This is a breaking change. Before, `Receiver<T>` implemented `Unpin` for all `T`. But now it only does it for `T: Unpin`... Does the message really need to implement `Unpin` for the receiver to do the same? :thinking:

Reply:
Good calling that out. I originally removed it since the receiver just contained a nonnull which is always Unpin, but now that the phantom data is there that's no longer the case so the impl needs to be re added